components/admin_console/team_channel_settings/channel/details/channel_groups.tsx

```diff
@@ -26,7 +26,7 @@ interface ChannelGroupsProps {
     isDisabled?: boolean;
 }
 
-export const ChannelGroups: React.SFC<ChannelGroupsProps> = (props: ChannelGroupsProps): JSX.Element => {
+export const ChannelGroups: React.FunctionComponent<ChannelGroupsProps> = (props: ChannelGroupsProps): JSX.Element => {
```

Comment:
This was a un-related deprecated function fix. (SFC is deprecated and need to use FunctionComponent)